packages/slate-editor/src/components/Input/Input.tsx

```diff
@@ -75,7 +75,9 @@ export const Input = forwardRef<HTMLInputElement, Props>(
         }, [value, pattern]);
 
         useEffect(() => {
-            setTimeout(() => input.current?.focus(), 0);
+            if (autoFocus) {
+                setTimeout(() => input.current?.focus(), 0);
+            }
```

Comment:
I think this might have been a bug as it was always focusing the last rendered input regardless of `autoFocus` value.

I think the reason it was never apparent was because we only render one input at a time.